docs/getting-started.txt

```diff
@@ -1,300 +1,311 @@
+   using the ``det deploy local`` command, the URL should be http://localhost:8080/.
 
-   This section requires a Determined cluster with multiple GPUs.
+#. Accept the default determined username with no password and click **Sign In** to view the
+   dashboard:
 
-Determined can coordinate multiple GPUs to train a single trial more quickly (*distributed
-training*). Distributed training performs best with complex models; therefore, the simple model used
-in this example may not demonstrate the full benefits of using distributed training.
+   .. image:: /assets/images/qs01b.png
+      :width: 704px
+      :align: center
+      :alt: Dashboard
 
-With Determined, moving from single-GPU training to distributed training is as simple as changing a
-single setting in the experiment config file. Specifically, :ref:`resources.slots_per_trial
-<exp-config-resources-slots-per-trial>` controls the number of GPUs used to train a single trial. It
-is ``1`` by default; setting it to a larger value enables distributed training. If
-``slots_per_trial`` is larger than the number of slots on any single agent machine, you should set
-``resources.slots_per_trial`` to a multiple of the number of GPUs in each machine in the cluster.
-For example, if your cluster has 8-GPU machines, you should use a value such as 8, 16, 24, etc. If
-using a Determined cluster deployed in the cloud, by default each agent will have eight GPUs.
+   In this example, you can see two experiments. Experiment 3 is in the COMPLETED state and
+   experiment 4 is in the ACTIVE state.
+
+   While an experiment is in the ACTIVE, training state, click an experiment tile to see the Metrics
+   graph update with your currently defined metrics. In this example, the graph displays the loss
+   and error rate per batch:
+
+   .. image:: /assets/images/qs04.png
+      :width: 704px
+      :align: center
+      :alt: Metrics graph detail
 
-Below is the ``distributed.yaml`` file, which is very similar to the ``const.yaml`` file except we
-now set ``resources.slots_per_trial``.
+#. When the experiment completes, click an experiment tile to view the trial page:
+
+   .. image:: /assets/images/qs03.png
+      :width: 704px
+      :align: center
+      :alt: Trial page
+
+   Because this experiment trains a single model with a fixed set of hyperparameters, there is only
+   one trial so selecting the experiment tile takes you directly to the trial page. [TBD: what does
+   this mean?]
+
+You can create additional experiments repeating the ``det experiment create`` command as a way to
+gain familiarity with other Determined WebUI functions.
+
+********************************
+ Run a Distributed Training Job
+********************************
+
+This exercise requires a Determined cluster with multiple GPUs.
+
+Determined can coordinate multiple GPUs to train a single trial more quickly by using distributed
+training. Distributed training is best applied to complex models. The model used in this example
```

Comment:
suggestion: drop the sentence on "best applied to complex models".

The question of when distributed training is "best applied" is slightly complicated; larger batch sizes through data parallel training do tend to perform worse past a certain point, and that point varies a lot based on other aspects of the model.  With the recent addition of DeepSpeed, Determined will support other types of parallelism beyond data parallel.  In any case, the introduction of cross GPU or cross machine communication overhead could in principle slow training down compared to single GPU if the communication channels are too slow.  I'm not sure this is well-summarized by "best applied to complex models".